### DISCUSSION TITLE
LiveSuit.sh not loading

### DISCUSSION THREAD

**Discussion by PainOfDeath** [Hypfer/Valetudo#2143] [Q&A / Support] [answered]
### Are you already using Valetudo?

No, but in the process of starting it

### Your questions

Hello,

I am trying to load Valetudo on a Dreame L10s Ultra using the Fastboot method as instructed. I don't have extensive Linux experience but was hoping to learn some more while doing this.

I did a clean install of Debian Bookworm on a laptop, using xfce, and ran the commands to install Livesuit per https://github.com/Hypfer/valetudo-sunxi-livesuit. Everything executed, except when finished, ./LiveSuit.sh returned "cannot connect to X server". Many searches for solutions yielded nothing that resolved it. Wondering if anyone else has encountered this, and hopefully has a solution or workaround.

If it matters, the laptop is a brand new Acer ChromeBook 311 that I repurposed for this project, following the steps and using firmware from mrchromebox.tech.

**Comment by vivia** [comment]
Seems to be essentially this problem https://askubuntu.com/questions/175611/cannot-connect-to-x-server-when-running-app-with-sudo

**Reply by PainOfDeath** [reply]
I just tried what was suggested as the solution in the article, still getting the same error.

**Reply by vivia** [reply]
How do you start your root shell?

**Reply by PainOfDeath** [reply, marked as answer]
Your question helped me to resolve this. I got it to work and it was the way I was starting the shell. Used "su" from a terminal this time, previously was using "su -" and was not aware of any difference. Thank you!

**Comment by The-Compiler** [comment]
> If it matters, the laptop is a brand new Acer ChromeBook 311 that I repurposed for this project

I believe that won't work out either way, as that's an ARM CPU, but the Livesuit binaries are x86 only.

**Reply by PainOfDeath** [reply]
I double-checked and this is an amd64 Chromebook.
